### ISSUE TITLE
last update  - tools disapperas (bug?)

### ISSUE THREAD

**Issue by Cronosus** [MJRLegends/ExtraPlanets#15]
ExtraPlanets-1.0-Pre 3.6

Hello
Carbon,Palladium and Magnesium tools disappears, but armors are still here.. something wrong? 
I have in config all armors/tools enabled.
I have some planets disabled, but i want all tools and armors enabled, because i have sources, for creating them, from other mods.

thanks and good work on this mod. ;)

**Comment by MJRLegends** [comment]
What planets do you have disabled and what tools were you still wanting

**Comment by Cronosus** [comment]
"main dimensions" {
    # Disabling this will remove the Jupiter & Tier 5 Rocket with all the related items/block!
    B:"Jupiter & Tier 5 Rocket"=false

    # Disabling this will remove the Neptune & Tier 8 Rocket with all the related items/block!
    B:"Neptune & Tier 8 Rocket"=false

    # Disabling this will remove the Pluto & Tier 9 Rocket with all the related items/block!
    B:"Pluto & Tier 9 Rocket"=false

    # Disabling this will remove the Saturn & Tier 6 Rocket with all the related items/block!
    B:"Saturn & Tier 6 Rocket"=false

    # Disabling this will remove the Uranus & Tier 7 Rocket with all the related items/block!
    B:"Uranus & Tier 7 Rocket"=true

    # Disabling this will remove the Venus & Tier 4 Rocket with all the related items/block!
    B:"Venus & Tier 4 Rocket"=false
}
"other dimensions" {
    B:Callisto=false
    B:Ceres=false
    B:Deimos=false
    B:Eris=true
    B:Europa=false
    B:Ganymede=false
    B:IO=false
    B:"Kepler SolarSystems"=true
    B:"Kuiper Belt"=false
    B:Makemake=false
    B:Mercury=false
    B:Phobos=false
    B:Rhea=false
    B:Titan=false
    B:Triton=false
}

i want all armors and tools

**Comment by MJRLegends** [comment]
```
# Disabling this will remove the Neptune & Tier 8 Rocket with all the related items/block!
B:"Neptune & Tier 8 Rocket"=false
# Disabling this will remove the Pluto & Tier 9 Rocket with all the related items/block!
B:"Pluto & Tier 9 Rocket"=false
# Disabling this will remove the Saturn & Tier 6 Rocket with all the related items/block!
B:"Saturn & Tier 6 Rocket"=false
# Disabling this will remove the Venus & Tier 4 Rocket with all the related items/block!
B:"Venus & Tier 4 Rocket"=false
```
The comment "all the related items/block!" is telling you this, as i changed it to do tools and that aswell and im going to stay with that change for the moment

**Comment by Cronosus** [comment]
ok, but armors still persists..

**Comment by MJRLegends** [comment]
Yeah that will also be changed to be removed to in the next update thats coming out in a min, please check commits after the one labelled "Released v1.0-Pre 3.6" or/and the change log
